drivers/serial/Kconfig.nrfx_uart_instance

```diff
@@ -106,11 +116,12 @@ config UART_$(nrfx_uart_num)_A2I_RX_SIZE
 config UART_$(nrfx_uart_num)_A2I_RX_BUF_COUNT
 	depends on !UART_NRFX_UARTE_LEGACY_SHIM
 	int "Asynchronous to interrupt driven adaptation layer RX buffer count"
-	default 8 if UART_$(nrfx_uart_num)_INTERRUPT_DRIVEN
+	default 4 if UART_$(nrfx_uart_num)_INTERRUPT_DRIVEN
```

Comment:
What is the impact of this change on existing users of this feature?

Reply:
there is not existing users yet as option is introduced by #65155 and 4 is actually better default number. I will use that in #65155 as well so there will be no diff here.

Reply:
@nordic-krch if I checked correctly, you did not include this change in https://github.com/zephyrproject-rtos/zephyr/pull/65155 after all. Should I include it here as a part of the alignment of UARTE driver to nRF54L15 or will it be included in some other PR regarding UARTE?

Reply:
i think you can drop this change and stick to what is in main.